getting_started/workflow/assets/importing_scenes.rst

```diff
@@ -269,7 +269,46 @@ Filter Script
 ~~~~~~~~~~~~~
 
 It is possible to specify a filter script in a special syntax to decide which tracks from which
-animations should be kept. (@TODO this needs documentation)
+animations should be kept.
+
+The filter script is executed against each imported animation. The syntax consists of two types of
+statements, the first for choosing which animations to filter, and the second for filtering
+individual tracks within the matched animation. All name patterns are performed using a case
+insensitive expression match, using `?` and `*` wildcards (using `String.matchn()` under the hood).
```

Comment:
Note that in reST, unlike Markdown, inline code is done with two backticks:
<pre>
``?``
</pre>